source/gui/addonStoreGui/viewModels/addonList.py

```diff
@@ -349,6 +351,20 @@ def setSortField(self, modelField: AddonListField):
 			# ensure calling on the main thread.
 			core.callLater(delay=0, callable=self.updated.notify)
 
+	def setColumnChoices(self) -> None:
+		# Translators: Presented when a list will be sorted in ascending order.
+		ascendingOrderLabel = pgettext("addonStore", "Ascending")
+		# Translators: Presented when a list will be sorted in descending order.
+		descendingOrderLabel = pgettext("addonStore", "Descending")
+		columnChoices = []
+		for c in self.presentedFields:
+			columnChoices.append(f"{c.displayString} ({ascendingOrderLabel})")
+			columnChoices.append(f"{c.displayString} ({descendingOrderLabel})")
+		self._columnChoices = columnChoices
+
+	def setReverse(self, reverse: bool) -> None:
+		self._reverse = reverse
```

Comment:
Instead of this function, update `setSortField` to take an optional `reverse` parame to set `self._reverseSort`.

```suggestion
```